src/pages/home/report/ReportActionsView.js

```diff
@@ -133,7 +132,7 @@ class ReportActionsView extends React.Component {
 
     shouldComponentUpdate(nextProps, nextState) {
         if (!_.isEqual(nextProps.reportActions, this.props.reportActions)) {
-            this.sortedAndFilteredReportActions = this.getSortedReportActionsForDisplay(nextProps.reportActions);
+            this.sortedAndFilteredReportActions = ReportActionsUtils.getSortedReportActionsForDisplay(nextProps.reportActions);
```

Comment:
Not-a-blocker: I really dislike seeing these side-effects in this method. It would be really nice to see if there is a way to move these somewhere else at some point. No need to do it in this PR, but if you have some ideas, it might be nice to open a GH to do this at some point.

Reply:
Thanks for the suggestion, I'll think about it separately from this PR.